src/components/v2/values/ea/EAValues.component.tsx

```diff
@@ -282,24 +336,14 @@ function ExternalAppValues({ appId }: { appId: string }) {
                                 installedAppInfo={installedAppInfo}
                                 releaseInfo={releaseInfo}
                             />
-                            {installedAppInfo && <StaticChartRepoInput releaseInfo={releaseInfo} />}
-                            {!installedAppInfo && (
-                                <ChartRepoSelector
-                                    isExternal={true}
-                                    installedAppInfo={installedAppInfo}
-                                    handleRepoChartValueChange={handleRepoChartValueChange}
-                                    repoChartValue={repoChartValue}
-                                    chartDetails={{
-                                        appStoreApplicationVersionId: 0,
-                                        chartRepoName: releaseInfo.deployedAppDetail.chartName,
-                                        chartId: 0,
-                                        chartName: releaseInfo.deployedAppDetail.chartName,
-                                        version: releaseInfo.deployedAppDetail.chartVersion,
-                                        deprecated: false,
-                                    }}
-                                />
-                            )}
-                            {!installedAppInfo && repoChartValue?.chartRepoName && (
+                            <ChartRepoSelector
```

Comment:
`{!installedAppInfo && ` this check is not required?

Reply:
Yes. It's not required anymore and other checks around `installedAppInfo` are already present in  ChartRepoSelector.